File: server/src/main/com/broll/gainea/server/core/bot/strategy/ICardStrategy.java
package com.broll.gainea.server.core.bot.strategy;

import com.broll.gainea.server.core.bot.CardOption;
import com.broll.gainea.server.core.bot.impl.BotSelect;

public interface ICardStrategy {

    CardOption strategy(BotStrategy strategy, BotSelect select);
}

File: server/src/main/com/broll/gainea/server/core/bot/strategy/ICardStrategy.kt
package com.broll.gainea.server.core.bot.strategy

import com.broll.gainea.server.core.bot.CardOption
import com.broll.gainea.server.core.bot.impl.BotSelect


interface ICardStrategy {
    fun strategy(strategy: BotStrategy, select: BotSelect): CardOption
}
